ngx-fudis/projects/ngx-fudis/src/lib/directives/button-base/button-base.directive.ts

```diff
@@ -30,17 +26,9 @@ export class ButtonBaseDirective
   extends PopoverApiDirective
   implements OnInit, OnDestroy, OnChanges
 {
-  constructor(
-    protected _idService: FudisIdService,
-    private _dropdownEventService: DropdownEventService,
-  ) {
+  constructor(protected _idService: FudisIdService) {
     super();
   }
-  /**
-   * Reference to native button element
-   */
-  @ViewChild('buttonElement') public buttonEl: ElementRef<HTMLButtonElement>;
```

Comment:
Seems like this is now in both Button and IconButton, so should/could we move it to back here?